tensorpack/graph_builder/distributed.py

```diff
@@ -330,6 +435,7 @@ def _apply_gradients_and_copy(self, opt, raw_grad_list, ps_var_grads):
                 apply_gradient_op = opt.apply_gradients([(g, v)])
                 barrier = self._add_sync_queues_and_barrier(
                     'param_update_barrier_{}'.format(vid), [apply_gradient_op])
+
```

Comment:
As a general rule, I would say to make sure to remove all these extraneous lines and the other unnecessary changes in spacing, though I'm not going to press much in this situation.